lib/vmpooler/janitor.rb

```diff
@@ -0,0 +1,39 @@
+module Vmpooler
+  class Janitor
+    def initialize
+      # Load the configuration file
+      config_file = File.expand_path('vmpooler.yaml')
+      $config = YAML.load_file(config_file)
```

Comment:
Agreed... I think there are likely several chunks of code that would be better suited as helper methods.  I think modifying all of the libraries would be out of scope for this PR, though?  I'd rather do it in it's own individual PR, at least; what do you think?